content/en/real_user_monitoring/error_tracking/_index.md

```diff
@@ -29,9 +29,9 @@ algolia:
 
 It is critical for your system's health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. By tracking, triaging, and debugging crashes, you can minimize the impact of fatal errors on your web and mobile applications' user experience.
 
```

Comment:
Should we mention somewhere here how to navigate to the Error Tracking page from UX Monitoring or elsewhere?